src/index.spec.js

```diff
@@ -2095,11 +2095,13 @@ describe('useDropzone() hook', () => {
     it('rejects all files if {multiple} is true and maxFiles is less than files and {accept} criteria is met', async () => {
       const onDropSpy = jest.fn()
       const onDropRejectedSpy = jest.fn()
-      const ui = (
-        <Dropzone accept="image/*" onDrop={onDropSpy} onDropRejected={onDropRejectedSpy} multiple={true} maxFiles={1}>
-          {({ getRootProps, getInputProps }) => (
+      const ui = ( 
+         <Dropzone accept="image/*" onDrop={onDropSpy} onDropRejected={onDropRejectedSpy} multiple={true} maxFiles={1}>
```

Comment:
The whitespace here is off:

```suggestion
      const ui = (
        <Dropzone accept="image/*" onDrop={onDropSpy} onDropRejected={onDropRejectedSpy} multiple={true} maxFiles={1}>
```